Procfile

```diff
@@ -1,3 +1,7 @@
-web: python manage.py serve -d -r -h 0.0.0.0 -p 6666
-worker: celery -A udata.worker worker
-beat: celery -A udata.worker beat
+# Uncomment lines given your setup and workflow:
```

Comment:
Better: provide a `Procfile.sample` and add `Procfile` to `.gitignore` to avoid bad commits.